Sources/Fog/View/FogView.swift

```diff
@@ -193,6 +193,14 @@ extension LedgerTxOut {
         -> Result<LedgerTxOut, ConnectionError>
     {
         guard let ledgerTxOut = LedgerTxOut(txOutRecord, viewKey: viewKey) else {
+            
```

Comment:
@the-real-adammork did you mean to double up the `guard let`s?

Reply:
no that was for testing